src/EditorFeatures/TestUtilities/NavigateTo/AbstractNavigateToTests.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.VisualStudio.Imaging.Interop;
 using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Language.NavigateTo.Interfaces;
+using Microsoft.VisualStudio.Text.PatternMatching;
 using Moq;
```

Comment:
this is base type of all these naviagetToTests included interactive navigate to tests. looks like you can define those static s_emptyxxxPatternMatches once here and reuse them in all other tests.

Reply:
Yes, I fixed that